### PR TITLE
AP_Mission: NFC Correct some comments

### PR DESCRIPTION
**Pull request by auturgy**
Some of the code comments were clearly C&P errors.  This fixes that.  I haven't corrected typos etc, to preserve history.

### CODE REVIEW

**Comment by rmackay9** [comment]
@auturgy, looking good, thanks!  Can you change the commit title so that it is "AP_Mission: NFC Correct some comments"?

**Comment by rmackay9** [comment]
@auturgy, Ah, I should have been more clear.  It's not the PR title but the commit title that should be fixed.  txs!

File: libraries/AP_Mission/AP_Mission.h [hunk 1]
```diff
@@ -244,7 +244,7 @@ class AP_Mission
         // do-guided-limits
         Guided_Limits_Command guided_limits;
 
-        // cam trigg distance
+        // high altitude balloon altitude wait
```
Comment (khancyr) [inline libraries/AP_Mission/AP_Mission.h:247]: why balloon?

Comment (tridge) [reply libraries/AP_Mission/AP_Mission.h:247]: it was originally added for balloon drops

**Comment by khancyr** [comment]
@tridge it need fixing on commit title

**Comment by peterbarker** [comment]
Commit message fixed in consultation with @auturgy

**Comment by rmackay9** [comment]
Merged, thanks!

**Pull request merged**